spec/helpers/journal_helper_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe JournalHelper do
+  describe '#time_range' do
+    let(:beginning_of_week) { Date.current.beginning_of_week }
+    let(:end_of_week) { Date.current.end_of_week }
+
+    it "is between the time range" do
+      range = beginning_of_week..end_of_week
+      expect(range).to cover Date.today
```

Comment:
I don't see you test helper methods
should be `expect(helper.time_rage).to eql(<expected_result>)`